go.mod

```diff
@@ -43,3 +43,5 @@ require (
 	google.golang.org/grpc v1.38.0
 	pgregory.net/rapid v0.4.0
 )
+
+replace github.com/livepeer/lpms => ../lpms
```

Comment:
Was this intentional? I think it's what's causing the build to break

Reply:
Yes, you are right. We usually pin LPMS version right before merge once PR is approved, so I didn't pay attention to `go.mod` for now.